examples/usecases/mcp_supabase_migration_agent/main.py

```
@@ -0,0 +1,146 @@
+                
+                You will work with a project located at {project_path} and process the migration file {migration_file}.
+                
+                Be careful not to overwrite or incorrectly merge existing type definitions. Ensure backward compatibility
+                and follow the project's code style for consistency.""",
+                server_names=["supabase", "github"],
+            )
+
+            try:
+                llm = await supabase_migration_agent.attach_llm(OpenAIAugmentedLLM)
+
+                prompt = f"""Complete the following workflow for automating Supabase migration code generation and GitHub commits:
```

Pylint (3.3.7)

[convention] Line too long (130/100)

(C0301)